ui/desktop/app/routes/scopes/scope/projects/settings/index.js

```diff
@@ -4,5 +4,39 @@
  */
 
 import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
 
-export default class ScopesScopeProjectsSettingsIndexRoute extends Route {}
+export default class ScopesScopeProjectsSettingsIndexRoute extends Route {
+  @service ipc;
+
+  async model() {
+    let cacheDaemonStatus,
+      formattedCacheVersion,
+      formattedCliVersion,
+      formattedDesktopVersion;
+    const { versionNumber: cliVersion } =
+      await this.ipc.invoke('getCliVersion');
+    formattedCliVersion = `v${cliVersion}`;
+
+    const { desktopVersion } = await this.ipc.invoke('getDesktopVersion');
+    formattedDesktopVersion = `v${desktopVersion}`;
+    try {
+      cacheDaemonStatus = await this.ipc.invoke('cacheDaemonStatus');
+    } catch (e) {
+      __electronLog?.warn('cache daemon is not running', e.message);
+      return {
+        formattedCliVersion,
+        formattedCacheVersion,
+        formattedDesktopVersion,
+        warning: true,
```

Comment:
I'm using this to show the warning alert in the UI, open to feedback on better ways to achieve this

Reply:
It might be better to return the actual error so we can show any error messages?

Reply:
good call, I'm returning the err obj, we will display the message in the UI if there's any, let me know what you think